docs/guides/context.md

```diff
@@ -69,15 +69,29 @@ In this scenario, `value.userId` helps retrieve the `userId` information from th
 
 ### env
 
-This field represents global environment variables for the server, set once when the server starts.
+Environment variables represent global configurations for the server. They are set once when the server starts. These variables can be utilized within directives to dynamically configure behavior based on server settings. This allows for flexible and adaptable API configurations that can be managed without hardcoding values into the schema.
+
+For example:
 
 ```graphql showLineNumbers
 type Query {
   users: [User]! @http(baseUrl: "{{env.API_ENDPOINT}}", path: "/users")
 }
 ```
 
-Here, `env.API_ENDPOINT` refers to an environment variable named API_ENDPOINT, defined in your server settings.
+Here, env.API_ENDPOINT refers to an environment variable named API_ENDPOINT, which is defined in your server settings. The value of API_ENDPOINT is used as the base URL for making HTTP requests
+
+### variables
+
+Variables enable a simple way to add dynamic values into GraphQL directives, providing flexibility and adaptability within the schema. Integration of variables into directives enables the creation of more flexible and reusable schema configurations.
+
+```graphql showLineNumbers
+type Query {
+  user(id: ID!) @http(url: "/users/{{args.id}}")
+}
+```
+
+Here, `args.id` is a variable representing the value of the id argument provided in the GraphQL query. During query execution, this variable is replaced with the actual id value, enabling the construction of dynamic URLs within the directive.
```

Comment:
variables should be set using `var` and not `args`. Create an example using `var`.